src/util/util.js

```diff
@@ -132,20 +132,55 @@ async function formatAuctionHistory(name, txs) {
 
   let history = [];
 
-  console.log(nameHash);
 
   //XXX Double check - but I believe that all name actions will be outputs
   for (let tx of txs) {
     let fulltx = await client.getTX(tx.tx_hash);
-    console.log(fulltx.inputs);
-    console.log(fulltx.inputs[0].coin.covenant);
-    console.log(fulltx.outputs);
+    let newtx = {};
     for (let o of fulltx.outputs) {
       let cov = new Covenant(o.covenant.type, o.covenant.items);
 
       if (cov.isName()) {
         if (cov.get(0) === nameHash) {
-          history.push(fulltx);
+          if (cov.isOpen()) {
```

Comment:
Hmmm if these are all mutually exclusive, I'd make these `else if`s just to be more clear, but probably doesn't warrant a change